### DISCUSSION TITLE
esp32 in deep sleep

### DISCUSSION THREAD

**Discussion by cruzer619** [jameszah/ESP32-CAM-Video-Telegram#18] [Q&A]
Would there be any issue adding deep sleep in your code.  Yes i'm aware that you won't be able to communicate with the esp32 via telegram if in deep sleep.  But once awake, should be able to send messages..

thanks

RC

**Comment by jameszah** [comment]
Saw this long ago, but could not find it again.
Yes, you could deepsleep and wake on event, take the avi clip, and then start the wifi, and the telegram, send the clip through, and respond to messages, then wait for a minute for new messages, and then deepsleep again.
I've thought of working on that -- but not at the top of the list.

**Reply by cruzer619** [reply]
ok thanks for getting back to me...  I tried to insert deepsleep from the deepsleep example but not working right.   after being woken up.. want it to stay on for about 10-15min then go back to sleep.  But just cycles sleep/wakeup/sleep/wakeup..  I thought I would give it try but my coding skills need work.

**Reply by 735751363r** [reply]
Hi, sorry, i´m a very beginner, so I can not help you in any way. But I want to use the ESP outdoors with an LiPo powered so deep sleep would be an essential feature. Have you made any Progress with the deep sleep programm. Or did you even managed it to work?
